lib/src/events/mixins.dart

```diff
@@ -137,6 +138,12 @@ mixin MouseSignalsMixin<T extends MouseInputData> {
   EventSignal<T> get onMouseOut => $onMouseOut ??= EventSignal();
   EventSignal<T> get onMouseScroll => $onMouseWheel ??= EventSignal();
 
+  EventSignal<T> onLongPress([Duration duration =
```

Comment:
Can we go back to this simpler way ? (with the defaults values)

```dart
obj.longPressDelay = 0.1; // in seconds
obj.longPressDistance = 20; // in points
obj.onLongPress.add( callback ) ;
```

Reply:
already back to this.